config/testConfig.js

```diff
@@ -3,6 +3,8 @@ import path from 'path'
 import chai from 'chai'
 import chaiAsPromised from 'chai-as-promised'
 
+process.env.APP_BASEURL = 'https://chat.learnersguild.org'
```

Comment:
I prefer not to use production urls in tests. Makes it that much easier for something to go wrong.